config/crd/bases/kuadrant.io_managedzones.yaml

```diff
@@ -0,0 +1,146 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
+  name: managedzones.kuadrant.io
+spec:
+  group: kuadrant.io
+  names:
+    kind: ManagedZone
+    listKind: ManagedZoneList
+    plural: managedzones
+    singular: managedzone
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - description: Domain of this Managed Zone
+      jsonPath: .spec.domainName
+      name: Domain Name
+      type: string
+    - description: The ID assigned by this provider for this zone .
+      jsonPath: .status.id
+      name: ID
+      type: string
+    - description: Number of records in the provider zone.
+      jsonPath: .status.recordCount
+      name: Record Count
+      type: string
+    - description: The NameServers assigned by the provider for this zone.
+      jsonPath: .status.nameServers
+      name: NameServers
+      type: string
+    - description: Managed Zone ready.
+      jsonPath: .status.conditions[?(@.type=="Ready")].status
+      name: Ready
+      type: string
+    name: v1
```

Comment:
maybe v1alpha1 here too as it's feels too soon to be locking down a version

Reply:
Changed to v1alpha1